src/mod/applications/mod_callcenter/mod_callcenter.c

```diff
@@ -3520,7 +3674,8 @@ static int list_result_json_callback(void *pArg, int argc, char **argv, char **c
 "\tcallcenter_config queue count | \n" \
 "\tcallcenter_config queue count agents [queue_name] [status] [state] | \n" \
 "\tcallcenter_config queue count members [queue_name] | \n" \
-"\tcallcenter_config queue count tiers [queue_name]"
+"\tcallcenter_config queue count tiers [queue_name] | \n" \
+"\tcallcenter_config agent set not_registered_delay_time [agent_name] [wait second] | \n"
```

Comment:
```suggestion
"\tcallcenter_config agent set not_registered_delay_time [agent_name] [wait second]"
```

Reply:
Also, instead of putting it at the end, group it with their sibling.  Wasn't there another type you added as well ?